.shared_rubocop.yml

```diff
@@ -50,6 +51,11 @@ Lint/RaiseException:
 Lint/StructNewOverride:
   Enabled: true
 
+Lint/RedundantDirGlobSort:
+  Enabled: true
+  Exclude:
+    - 'spec/rails_helper.rb'
```

Comment:
this has some boilerplate that we don't want to change even though it fails rubo